### PR TITLE
HWKAPM-619 Fix thread leak when looping waiting for collector configu…

### PR DESCRIPTION
**Pull request by objectiser**
…ration

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Can you review please?

Related to: https://issues.jboss.org/browse/HWKAPM-619

File: client/instrumenter/src/main/java/org/hawkular/apm/instrumenter/ClientManager.java [hunk 1]
```diff
@@ -102,24 +132,14 @@ protected static void initConfig() {
...
-                }
-            }).schedule(new Runnable() {
-                @Override
-                public void run() {
-                    initConfig();
-                }
-            }, configRetryInterval, TimeUnit.SECONDS);
+
+            // Shutdown the executor
+            configExecutor.shutdownNow();
```
Comment (pavolloffay) [inline client/instrumenter/src/main/java/org/hawkular/apm/instrumenter/ClientManager.java]: Is this call necessary? It is called on line 109.

Comment (objectiser) [inline client/instrumenter/src/main/java/org/hawkular/apm/instrumenter/ClientManager.java]: Missed this one - will remove. :+1:

**Comment by pavolloffay** [comment]
LGTM :+1:

**Pull request merged**